core/src/main/java/uk/gov/justice/services/core/accesscontrol/AccessControlService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.services.core.accesscontrol;
+
+import uk.gov.justice.services.messaging.JsonEnvelope;
+
+import java.util.Optional;
+
+import static java.util.Optional.empty;
+
+public class AccessControlService {
+
+    public static final String ACCESS_CONTROL_DISABLED_PROPERTY =
+                    "uk.gov.justice.services.core.accesscontrol.disabled";
+
+    private final PolicyEvaluator policyEvaluator;
+
+    public AccessControlService() {
+        this(new AllowAllPolicyEvaluator());
+    }
+
+    public AccessControlService(PolicyEvaluator policyEvaluator) {
```

Comment:
We should use our CDI convention here - the PolicyEvaluator should be package scoped and injected with @Inject, and no constructor. We will create a PolicyEvaluatorProducer to inject it, I think.